packages/website/pages/_app.tsx

```diff
@@ -1,12 +1,58 @@
 import React from 'react';
 import type { AppProps } from 'next/app';
-import { GlobalStyles } from '@contentful/f36-components';
+import Head from 'next/head';
+import { css } from 'emotion';
+import { GlobalStyles, Grid } from '@contentful/f36-components';
+
+const styles = {
+  grid: css({
+    height: '100vh',
+    overflow: 'hidden',
+    gridTemplateAreas: `"topbar topbar"
+    "sidemenu content"`,
+  }),
+  gridItem: css({
+    display: 'flex',
+    flexDirection: 'column',
+    overflow: 'auto',
+  }),
+  footer: css({
+    position: 'sticky',
+    top: '100vh',
+  }),
+};
 
 function MyApp({ Component, pageProps }: AppProps) {
   return (
     <>
       <GlobalStyles />
-      <Component {...pageProps} />
+      <Head>
+        <title>Forma 36 - The Contentful Design System</title>
+        <meta
+          name="description"
+          content="Forma 36 - The Contentful Design System"
+        />
+        <link rel="icon" href="/favicon.png" />
+      </Head>
+
+      <Grid
```

Comment:
I think it would be good to move it to `website/components/Layout.tsx`

Reply:
What exactly? The Grid wrapping the Grid.Items?

I thought this way would be easier to understand what is happening
easier than:
```
<Layout>
  <Grid.Item>Topbar</Grid.Item>
  <Grid.Item>Sidemenu</Grid.Item>
  <Grid.Item>Content</Grid.Item>
</Layout>
```

Reply:
Or are you suggesting to also move `Grid.Item` to separate components like `Topbar`, `Sidemenu`, etc.?

Reply:
I'd only leave `<Layout><Component {...pageProps} /></Layout>`

Reply:
Because at some point we'll probably have per page layouts: https://nextjs.org/docs/basic-features/layouts#per-page-layouts

Reply:
As this website grows out of only `Forma36` documentation and will have `field-editors` section and so on.

Reply:
I don't get it, this is the same structure we have right now
the content of the pages have different layouts you can see it by comparing the Homepage, "Getting started guide", and any other component page

They use a "Layout" component but those pages still fall into the "content" area

This PR is just making sure that we will render the things around those content even when we change pages

And if we really feel the necessity to use `getLayout` in the future we do it in the future
no need to solve a problem we do not have now

Reply:
Let me push this abstraction and you tell me If I understood the suggestion correctly or if I'm missing something